src/vm/fieldmarshaler.h

```diff
@@ -795,10 +808,38 @@ class FieldMarshaler_NestedValueClass : public FieldMarshaler
         return m_pNestedMethodTable.GetValueMaybeNull();
     }
 
+    BOOL IsFixedBuffer() const
+    {
+        return m_numFixedBufferElements != 0;
```

Comment:
Isn't a length of 0 valid for a fixed buffer? This might be better make -1.

Reply:
Will do